tools/c7n_azure/c7n_azure/function_package.py

```diff
@@ -28,6 +27,8 @@
 from c7n.mu import PythonPackageArchive
 from c7n.utils import local_session
 
+from c7n_azure.dependency_manager import DependencyManager
```

Comment:
nit: I forget half the time, but if you `ctrl+alt+o` in PyCharm it'll auto organize these for you so modules are together and alphabetized (so there isn't a c7n_azure in the middle and one at the bottom).